### PR TITLE
Undefined name: import sys for line 188

### PR DESCRIPTION
**Pull request by cclauss**
#### CheckList
- [ ] Did you add an unittest for your new example or defect fix?
- [ ] Did you add documents for your new example?
- [ ] All CIs are green? (You can check it after submitting)

### CODE REVIEW

File: PathPlanning/RRT/rrt_with_sobol_sampler.py [hunk 1]
```diff
@@ -28,6 +28,7 @@
 
 import math
 import random
+import sys
```
Comment (AtsushiSakai) [inline PathPlanning/RRT/rrt_with_sobol_sampler.py:31]: Thank you for your PR. Is it possible to add a test to run this script?

Comment (cclauss) [reply PathPlanning/RRT/rrt_with_sobol_sampler.py:31]: https://github.com/AtsushiSakai/PythonRobotics/blob/master/tests/test_rrt_with_sobol_sampler.py

Comment (AtsushiSakai) [reply PathPlanning/RRT/rrt_with_sobol_sampler.py:31]: OK…. There is, I’m no sure why the test did not detect the missing import.

Comment (AtsushiSakai) [reply PathPlanning/RRT/rrt_with_sobol_sampler.py:31]: OK. This code will be used only when path drawing. Make sense

**Review by AtsushiSakai — APPROVED** [review]
LGTM. Thanks

**Pull request merged**